gui-js/apps/minsky-electron/src/app/managers/CommandsManager.ts

```
@@ -1341,10 +1342,14 @@
     let clientId='I9sn5lKdemBdh8uTNA7H7YiplxQk3gI-pP0I9_2g1tcbE88T2C3Z9wOvoy51I4-U';
     // need to pass what platform we are
     switch (process.platform) {
-    case 'win32': var state={system: 'windows', distro: '', version: '', arch:''}; break;
-    case 'darwin': var state={system: 'macos', distro: '', version: '', arch: `${process.arch}`}; break;
+    case 'win32':
+      var state={system: 'windows', distro: '', version: '', arch:'', previous: ''};
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)